projects/js-toolkit/packages/js-toolkit-core/src/project/liferayCli/Build.ts

```diff
@@ -50,6 +58,11 @@ export default class Build {
 					config as CustomElementBuildConfig
 				);
 				break;
+			case 'themeSpritemap':
```

Comment:
I thought prettier enforced lines above `case` (as in `return`) :thinking: 

If, not, it's incredible how consistent I've managed to be :joy: